src/test/java/org/reaktivity/specification/http/internal/FunctionsTest.java

```diff
@@ -56,11 +59,23 @@ public void headerTest()
         byte[] expected = new byte[] { (byte) 0x00, (byte) "name".length(), (byte) 'n', (byte) 'a', (byte) 'm', (byte) 'e',
                 (byte) "value".length(), (byte) 'v', (byte) 'a', (byte) 'l', (byte) 'u', (byte) 'e' };
         byte[] actual = header("name", "value");
+        DirectBuffer header = new UnsafeBuffer(actual);
+        HttpHeaderFW fw = new HttpHeaderFW().wrap(header, 0, actual.length);
+        Assert.assertEquals(fw.representation(), 0x00);
+        Assert.assertEquals(fw.name().asString(), "name");
+        Assert.assertEquals(fw.value().asString(), "value");
+
         Assert.assertArrayEquals(expected, actual);
 
```

Comment:
Split this into two tests, but only if you are testing something different in each test, otherwise a single test is sufficient.